remote/style.css

```diff
@@ -97,13 +97,100 @@ body > .is-root-container,
 	 padding-left: var(--wp--custom--spacing--outer);
  }
 
+/* Button styles */
+.wp-block-search__button,
+.wp-block-button__link,
```

Comment:
Do we need this line?

Reply:
Yep, for example to target the buttons added in the editor using "Buttons" block.

Reply:
I think they are controlled by the theme.json file, so they don't need to be targetted here.

Reply:
Did you remove the right line? I thought it was `wp-block-button__link` we should remove...

Reply:
I think so

Reply:
I think so